src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -41,23 +43,32 @@ func newBootstrapCmd() *cobra.Command {
 				return apiv1.ErrAlreadyBootstrapped
 			}
 
-			config := apiv1.BootstrapConfig{}
+			bootstrapConfig := apiv1.BootstrapConfig{}
+			if bootstrapCmdOpts.interactive && bootstrapCmdOpts.configFile != "" {
```

Comment:
`bootstrapCmdOpts.configFile` is `init.yaml` by default.
This means:
```
sudo k8s bootstrap --interactive
``` 
already triggers this error, right?